helm/templates/statefulset.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: edc
+  labels:
+    app.kubernetes.io/name: edc
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: edc
+  serviceName: edc
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: edc
+    spec:
+      containers:
+        - name: edc
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: "{{ .Values.image.pullPolicy }}"
+          ports:
+            - containerPort: {{ .Values.container.port }}
+              name: http
+
+          readinessProbe:
+            initialDelaySeconds: 1
```

Comment:
is there a particular reason to override the default initialDelaySeconds values?

Reply:
I kept the same config as the one specified in the README of the observability extension.